.github/maintainers_guide.md

```diff
@@ -71,25 +71,58 @@ If you make a mistake, don't fret. NPM allows you to unpublish a release within
     - bump the version in `package.json` for all packages you are releasing
     - Make a single commit for the version bump. The commit message should be similar to a [previous one](https://github.com/slackapi/node-slack-sdk/commit/1503609d79abf035e9e21bad7360e124e4211594).
     - Add version tags for each package you are about to release. Ex `git tag @slack/web-api@5.6.0`
-    - Push commit and tags up to origin: `git push origin master && git push origin --tags`. 
+    - Push commit and tags up to your fork: `git push username main --tags`. 
+    - Create pull request for review. It should be similar to a [previous one](https://github.com/slackapi/node-slack-sdk/pull/1059)
 
-2. Publish the release(s) to npm: `NPM_CONFIG_OTP=xxxxxx npx lerna publish from-package`
-  * You should have [2FA set up with NPM](https://docs.npmjs.com/about-two-factor-authentication), and the
+2.  Merge into main repository
+  *  Once tests pass and a reviewer has approved, merge the pull request. You will also want to
+    update your local `main` branch.
```

Comment:
Perhaps add a separate bullet for "You will also want to update your local `main` branch." such as:
- Update your local `main` branch `git rebase origin main` (or `git pull origin main`)